<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>carsearch</class>
 <widget class="QDialog" name="carsearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>531</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>531</width>
     <height>381</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>차량이름</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>유종</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>특징</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>대여가능여부</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>금액</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>랜트카</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>차량이름</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>유종</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>특징</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>540</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>540</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>를 선택하였습니다.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>590</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>돌아가기</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="search">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>80</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>검색</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reserv">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>590</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>예약하기</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
